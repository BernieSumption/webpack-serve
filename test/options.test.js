// const { resolve } = require('path');

const { getOptions } = require('../lib/options');

describe('options', () => {
  test('defaults', () => {
    const argv = {};
    const opts = {};
    return getOptions(argv, opts).then((result) => {
      expect(result).toMatchSnapshot();
    });
  });
});
