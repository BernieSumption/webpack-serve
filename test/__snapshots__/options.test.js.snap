// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`options defaults 1`] = `Promise {}`;
